README.md

```diff
@@ -1,113 +1,125 @@
+* Provide one `etcd` proxy (`etcd-mesos-proxy`) and optional SRV lookup support via [Mesos-DNS](https://github.com/mesosphere/mesos-dns).
+
+For more information, please consult the following design documents:
 * [Architecture](docs/architecture.md)
 * [Administration](docs/administration.md)
 * [Incident Response](docs/response.md)
 
-## features
+## Deployment
 
-- [x] runs, monitors, and administers an etcd cluster of your desired size
-- [x] recovers from n/2-1 failures by reconfiguring the etcd cluster and launching replacement nodes
-- [x] recovers from up to n-1 simultaneous failures by picking a survivor to re-seed a new cluster (ranks survivors by raft index, prefering the replica with the highest commit)
-- [x] etcd proxy configurer (etcd-mesos-proxy) and optional SRV record support via mesos-dns
+There are quite a few different ways to run this framework but for the sake of simplicity let's deploy it as a [Marathon application](https://mesosphere.github.io/marathon/docs/application-basics.html):
 
-## running
-
-Marathon spec:
-```
+```json
 {
-  "id": "etcd",
-  "container": {
-    "docker": {
-      "forcePullImage": true,
-      "image": "mesosphere/etcd-mesos:0.1.0-alpha-target-23-24-25"
+  "id":"etcd",
+  "container":{
+    "docker":{
+      "image":"mesosphere/etcd-mesos:0.1.3",
+      "forcePullImage":true
     },
-    "type": "DOCKER"
-  },
-  "cpus": 0.2,
-  "env": {
-    "FRAMEWORK_NAME": "etcd",
-    "WEBURI": "http://etcd.marathon.mesos:$PORT0/stats",
-    "MESOS_MASTER": "zk://master.mesos:2181/mesos",
-    "ZK_PERSIST": "zk://master.mesos:2181/etcd",
-    "AUTO_RESEED": "true",
-    "RESEED_TIMEOUT": "240",
-    "CLUSTER_SIZE": "3",
-    "CPU_LIMIT": "1",
-    "DISK_LIMIT": "4096",
-    "MEM_LIMIT": "2048",
-    "VERBOSITY": "1"
+    "type":"MESOS",
   },
-  "healthChecks": [
+  "cpus":0.2,
+  "mem":128.0,
+  "instances":1,
+  "portDefinitions":[0, 0, 0],
+  "healthChecks":[
     {
-      "gracePeriodSeconds": 60,
-      "intervalSeconds": 30,
-      "maxConsecutiveFailures": 0,
-      "path": "/healthz",
-      "portIndex": 0,
-      "protocol": "HTTP"
+      "gracePeriodSeconds":60,
+      "intervalSeconds":30,
+      "maxConsecutiveFailures":0,
+      "path":"/healthz",
+      "portIndex":0,
+      "protocol":"HTTP"
     }
   ],
-  "instances": 1,
-  "mem": 128.0,
-  "ports": [
-    0,
-    1,
-    2
-  ]
+  "env":{
+    "FRAMEWORK_NAME":"etcd",
+    "FRAMEWORK_FAILOVER_TIMEOUT_SECONDS":"300",
+    "WEBURI":"http://etcd.marathon.mesos:$PORT0/stats",
+    "CLUSTER_SIZE":"3",
+    "MESOS_MASTER":"zk://localhost:2181/mesos",
+    "ZK_PERSIST":"zk://localhost:2181/etcd-mesos",
+    "VERBOSITY":"3",
+    "SINGLE_INSTANCE_PER_SLAVE":"true",
+    "AUTO_RESEED":"true",
+    "RESEED_TIMEOUT":"240",
+    "DISK_LIMIT":"1024",
+    "CPU_LIMIT":"0.2",
+    "MEM_LIMIT":"256"
+  }
 }
 ```
 
-## building
+## Build (optional)
```

Comment:
just split this section into two subsections. that'll make the requirements easier to specify.
- Build with Docker
- Build locally